### ISSUE TITLE
`cfg` in RLHFdataset doesn't have `length_params`

### ISSUE THREAD

**Issue by joonkeekim** [NVIDIA/NeMo-Aligner#173]
**Describe the bug**

In the line 39 and 40 in nemo_aligner/data/nlp/datasets.py [[link]](https://github.com/NVIDIA/NeMo-Aligner/blob/main/nemo_aligner/data/nlp/datasets.py#L39), `cfg` should be `cfg.ppo` as below
```
# if "length_params" in cfg:
if "length_params" in cfg.ppo:
```
```
# max_sample_length = seq_length - cfg.length_params.max_length
max_sample_length = seq_length - cfg.ppo.length_params.max_length
```
Since `length_params` is in cfg.model.ppo

**Comment by odelalleau** [comment]
Thanks for pointing this out! Should be fixed by #177

**Comment by joonkeekim** [comment]
Thank you for the quick processing! I will close the issue
